mmv1/products/firebasehosting/api.yaml

```diff
@@ -64,4 +64,67 @@ objects:
       - !ruby/object:Api::Type::String
         name: defaultUrl
         output: true
-        description: The default URL for the site in the form of https://{name}.web.app
\ No newline at end of file
+        description: The default URL for the site in the form of https://{name}.web.app
+  - !ruby/object:Api::Resource
+    name: 'Channel'
+    min_version: beta
+    base_url: projects/{{project}}/sites/{{site_id}}/channels
```

Comment:
Base url probably shouldn't use project if it isn't used during create or any actual calls

Reply:
I've tried omitting the {{project}} field. However, this seems to mean I can no longer use the `project` field in the terraform config, and therefore can't specify a billing project used for the API call. Is there another way to specify the project to bill?
FWIW, one of the valid import formats is  `'projects/{{project}}/sites/{{site_id}}/channels/{{channel_id}}'` (see terraform.yaml).

Reply:
I think we should target using a field called `billing_project` for this instead. See a similar field here https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloud_asset_organization_feed#billing_project

Reply:
That makes sense, on import we wouldn't have access to the value set in the config.

Do we need to have the billing project set to be able to query the resource?

Reply:
Yes, we need a billing project to authenticate the user (in the case of tests, my application default credentials). The strange thing is, why imports use the project specified in `projects/{{project}}`, but not `billing_project` even when configured? Does `projects/{{project}}` generate some magic parameter that gets passed through everything?

Reply:
Added a commit to use the `billing_project` field so we can see the error here

Reply:
Does the import action only work when the environment variable is set? We could do something similar to the cloudasset resources that use a custom import to only set the name to the import ID: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/templates/terraform/custom_import/cloud_asset_feed.go.erb

Reply:
Yes it only works when the environment variable is set. I've tried only setting the name like [cloud_asset_feed.go.erb](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/templates/terraform/custom_import/cloud_asset_feed.go.erb) but failed with missing site_id and channel_id, so I think at least the generated import code was correct there.

I believe we should use the environment variable `GOOGLE_BILLING_PROJECT` because:
1. [Documentation](https://github.com/GoogleCloudPlatform/magic-modules/blob/eb6ac6344b4f99ef5d75932e98d6f0ffea1cfc16/mmv1/third_party/terraform/website/docs/guides/provider_reference.html.markdown) says so 
2. The integration tests passed, meaning the environment variable is set there.
3. `project` actually behaves the same way during import, in particular, the value is supplied form environment variable https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/utils/import.go#L80

Alternatively, we can introduce a fallback to set `billing_project` to `project`, but not sure if it's recommended.

Reply:
I suspect that we don't need a `billing_project` field at all, since it should be either in the environment or set as part of the provider. Added a commit passing locally, see if tests are happy here.